is_mainmenu.php

```diff
@@ -32,11 +33,17 @@ public function __construct()
         $this->bootstrap = true;
         parent::__construct();
 
-        $this->displayName = 'Main menu module';
-        $this->description = 'Main menu module';
+        $this->displayName = $this->trans('Main menu module', [], TranslationDomains::TRANSLATION_DOMAIN_ADMIN);
+        $this->description = $this->trans('Main menu module', [], TranslationDomains::TRANSLATION_DOMAIN_ADMIN);
```

Comment:
Does the translation domain loaded from a different file work when you export translation for a module? i.e. if you want to build a release

Reply:
@kpodemski it have no idea tbh 🤔. I will test it when everything is ready.